inc/class-wp-sitemaps-renderer.php

```diff
@@ -198,12 +205,12 @@ public function get_sitemap_xml( $url_list ) {
 		foreach ( $url_list as $url_item ) {
 			$url = $urlset->addChild( 'url' );
 
-			// Add each attribute as a child node to the URL entry.
```

Comment:
This change isn't necessary because it's handled in #201 (https://github.com/GoogleChromeLabs/wp-sitemaps/pull/201/files#diff-cc90739d8286d058f89eaf22de6cec61L201)

Reply:
Not a big deal, will be resolved with the merge